internal/monitors/kubernetes/cluster/metrics/deployments.go

```diff
@@ -13,10 +15,10 @@ import (
 
 func datapointsForDeployment(dep *v1beta1.Deployment) []*datapoint.Datapoint {
 	dimensions := map[string]string{
-		"metric_source":        "kubernetes",
-		"kubernetes_namespace": dep.Namespace,
-		"uid":             string(dep.UID),
-		"kubernetes_name": dep.Name,
+		"metric_source":              "kubernetes",
+		"kubernetes_namespace":       dep.Namespace,
+		"kubernetes_deployment_uid":  string(dep.UID),
+		"kubernetes_deployment_name": dep.Name,
```

Comment:
Change this back to `kubernetes_name` since we don't need to make it so unique (i.e. to sync properties to)